.travis.yml

```diff
@@ -31,13 +32,16 @@ script:
   - npm install
 
   # Install python packages
-  - pip3 install numpy scipy matplotlib pandas sympy lxml chevron
+  - pip3 install -r requirements.txt
```

Comment:
Also make this change to `Dockerfile` and the ansible scripts? Isn't having 4 different environments awesome :-)

Reply:
I updated the `Dockerfile` and submitted a PR over in ansible-pl 🙂

Reply:
Sweet!